Src/Sentry.Xamarin/Internals/NativeIntegration.droid.cs

```diff
@@ -54,6 +54,14 @@ private void Platform_ActivityStateChanged(object sender, ActivityStateChangedEv
                     ["screen"] = _xamarinOptions.PageTracker?.CurrentPage,
                     ["state"] = e.State.ToString()
                 }, level: BreadcrumbLevel.Info);
+            if (e.State == ActivityState.Paused)
+            {
+                _xamarinOptions.SessionLogger?.StatePaused();
+            }
+            else if (e.State == ActivityState.Resumed)
```

Comment:
There are specific states related to pausing and resuming, for example"
state: Started
state: Resumed

and 
state: Stopped
state: Destroyed

You only need to check one of them since they always appear during pausing/resuming